src/@types/tiny-invariant/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare function invariant(
+  testValue: any,
+  format?: string,
+  ...extra: any[]
+): void;
+
+declare module 'tiny-invariant' {
+  export = invariant;
+}
```

Comment:
This is not available at `@types` (only the standard invariant). We could contribute this back. The API will never change.

Reply:
FYI: if a library already exports typings, there is no need for that to be available at definitely typed. I read in the Typescript docs that we should only look for the `@types` in case the library does not have typings packaged together.

Reply:
Btw: is it a convention to put this under `src/@types/...`? Because we also have `types/index.d.ts`.

Reply:
Yeah that's my understanding too. Packaged types, over `@types`, over our own types. However it didn't have any types packaged to what TS told me.

Reply:
For example, Formik puts those in `types/index.d.ts`

https://github.com/jaredpalmer/formik/blob/master/types/index.d.ts

So I did the same and assumed this was the convention.

Reply:
Yes to what I read it evolved as one recently. TypeScript will pick those up automatically, which is what I liked. Formik needs to set it to be included. https://github.com/jaredpalmer/formik/blob/master/tsconfig.json#L3